Makefile

```diff
@@ -112,26 +118,6 @@ check: dependencies
 	@echo "Checking for incorrect conditional trait interface overrides..."
 	@mono --debug OpenRA.Utility.exe all --check-conditional-trait-interface-overrides
 
-
-NUNIT_CONSOLE := $(shell test -f thirdparty/download/nunit3-console.exe && echo mono thirdparty/download/nunit3-console.exe || \
-	which nunit3-console 2>/dev/null || which nunit2-console 2>/dev/null || which nunit-console 2>/dev/null)
-nunit: core
```

Comment:
Why remove this completely? I got it working easily.

Reply:
I can push the commit with my changes.

Reply:
I requested / suggested removing this to drop unnecessary complexity and improve parity between the makefile / make.ps1 / "normal" c# project expectations. IMO we should be emphasizing running the tests through the more familiar IDE UI.

Hardcoding the `~/.nuget` location here also didn't sit right because we know that this path can be overridden by the user, and I expect downstream packages are likely to do this.

Reply:
For that last point, it would be possible to sniff nunit3-console like I did before; however, I can run `nunit3-console --noresult OpenRA.Test.dll` with the nunit3-console AUR package installed or load a supplemental alias into a shell session `alias nunit='nunit3-console --noresult OpenRA.Test.dll'`. The first point is quite reasonable, maintaining another feature on all three build chains multiplies the work.

"More familiar IDE UI" certainly is a subjective statement. I use IDEs as just one subset of tools, generally with better code navigation and semantic refactoring. I find `vim` useful for quick editing, especially after `git status` or `git diff`. `sed` one-liners followed by `git diff` are handy for situations like #17957. I do almost all version control, building and testing in a terminal. In fact, I almost always have at least one windows of terminals on my computer. That said, I realize that most probably use an IDE for almost every aspect of development.

Given that there are only a few of us that would need to run the unit tests and we can manage without `make nunit`, removing it appears to be the better option.